src/Administration/Resources/app/administration/src/module/sw-media/component/sidebar/sw-media-quickinfo/index.js

```diff
@@ -359,5 +365,21 @@ export default {
 
             this.$emit('update:item', { ...this.item, ...newItemConfig });
         },
+
+        runAppAction(action) {
+            if (typeof action.callback !== 'function') {
+                return;
+            }
+
+            const { fileName, mimeType, fileSize, url, id } = this.item;
+
+            action.callback({
```

Comment:
```suggestion
            action.callback(this.item);
```

Is there a need to limit the fields?

Reply:
<img width="602" height="177" alt="Image" src="https://github.com/user-attachments/assets/1bc5a151-c692-492f-9c30-568ea3f0fb1b" />

I limit properties because of this error occurs in callback in App 🤔